guild/ipy.py

```diff
@@ -438,10 +438,10 @@ def _run(op, flag_vals, opts, extra_attrs=None):
                 result = op(**flag_vals)
     except KeyboardInterrupt as e:
         exit_status = exit_code.KEYBOARD_INTERRUPT
-        util.raise_from(RunTerminated(run, e), e)
+        raise RunTerminated(run, e) from e
```

Comment:
I've been kind of just doing things in the general direction of getting away from python 2. I will plan on extracting this kind of thing from this PR before it goes in. It should be in a dedicated PR.